src/modules/ffmpeg/producer/av_producer.cpp

```diff
@@ -774,6 +774,11 @@ struct AVProducer::Impl
         if (buffer_.empty() || (frame_flush_ && buffer_.size() < 4)) {
```

Comment:
frame_flush_ is true after a seek. How does your change affect accuracy of frame_time_ around a seek?

Reply:
I haven't tested the real world effect in that case; Perhaps it's better to expand if condition to exclude in this case?

However I do believe it would be better for my patch to compare to duration_ as duration might be different (less) and we might overflow time a bit more in that case. 

Unless overflowing is wanted behavior (i.e.: we keep increasing frame_time_ with duration as long as we keep playout stills, which is what old (2.1) caspar did.

What do you think?